backend/src/main/java/de/laura/project/model/TriviaPointSummary.java

```diff
@@ -14,10 +17,9 @@
 
 public class TriviaPointSummary {
 
-    private int category;
-    private int amount;
-    private String difficulty;
-    private int points;
+    @Id
+    private String user;
+    private List<TriviaPointCategory> triviaPointCategory;
```

Comment:
I think if you use the "user" here as the ID, there can't be two users in the app with the same name.
I gues you could leave the @ID annotation, then the MongoDB creates a own unique ID?